frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -14,7 +14,7 @@ export class StorageClassDropdownInner extends React.Component<
   readonly state: StorageClassDropdownInnerState = {
     items: {},
     name: this.props.name,
-    selectedKey: null,
+    selectedKey: this.props.selectedKey,
```

Comment:
What would happen if this prop were to start with undefined and later change to something else?

Reply:
The behavior would be to show default class , and if that not present, show none